src/components/Generator/Generator.tsx

```diff
@@ -7,12 +7,13 @@ import styles from './generator.module.css';
 import { useForm } from 'react-hook-form';
 import Settings from './Settings';
 import GeneratedTaskfile from './GeneredTaskfile';
-import Form from "@/components/Form";
-import {Font} from "./GeneredTaskfile/buildHeader";
+import Form from '@/components/Form';
+import { Font } from './GeneredTaskfile/buildHeader';
 
 export type GeneratorSettings = {
 	project: string;
 	font: Font;
+	runtime: 'local' | 'docker-compose' | 'docker-compose-dev';
```

Comment:
```suggestion
	runtime: 'local' | 'docker-compose';
```

Make a separate checkbox for including the development proxy when `docker-compose` is chosen